src/nfa/nfa.cc

```diff
@@ -2238,8 +2288,15 @@ Move& Move::operator=(Move&& rhs) noexcept {
 }
 
 void Move::insert(State s) {
-    if (targets.find(s) == targets.end()) {
-        targets.insert(s);
+    if(targets.empty() || targets.back() < s) {
+        targets.push_back(s);
+        return;
+    }
+    // Find the place where to put the element (if not present).
+    // insert to OrdVector without the searching of a proper position inside insert(const Key&x).
```

Comment:
Is this comment at the right place?

Reply:
I think so. It is just a multiline comment related the following code.

Reply:
Ok. It is confusing because it says insert without searching and the next line is searching.